docs/linux/fs.md

```diff
@@ -6,127 +6,135 @@ sidebar_label: File Systems
+device itself.
 
 The super block is always written on disk at sector 0. The first zone of the
-device storing the super block is never exposed as a zone file by *zonefs*. If
-the zone containing the super block is a sequential zone, the `mkzonefs` format
-tool always "finishes" the zone, that is, it transitions the zone to a full
-state to make it read-only, preventing any data write.
+device that stores the super block is never exposed as a zone file by
+*zonefs*. If the zone that contains the super block is a sequential zone, the
+`mkzonefs` format tool always "finishes" the zone (that is, it transitions the
+zone to a full state to make it read-only, preventing any data write).
 
 ### Zone Type Sub-Directories
 
-Files representing zones of the same type are grouped together under the same
-sub-directory automatically created on mount.
+Files that represent zones of the same type are grouped together under the same
+sub-directory, which is automatically created on mount.
 
 For conventional zones, the sub-directory "cnv" is used. This directory is
-however created if and only if the device has usable conventional zones. If
-the device only has a single conventional zone at sector 0, the zone will not
-be exposed as a file as it will be used to store the *zonefs* super block. For
-such devices, the "cnv" sub-directory will not be created.
+created only if the device has usable conventional zones. If the device has
+only a single conventional zone at sector 0, the zone will not be exposed as a
+file (because it will be used to store the *zonefs* super block). For such
+devices, the "cnv" sub-directory will not be created.
 
 For sequential write zones, the sub-directory "seq" is used.
 
 These two directories are the only directories that exist in *zonefs*. Users
-cannot create other directories and cannot rename nor delete the "cnv" and
-"seq" sub-directories.
+cannot create other directories and can neither rename nor delete the "cnv"
+and "seq" sub-directories.
 
-The size of the directories indicated by the `st_size` field of `struct stat`,
-obtained with the `stat()` or `fstat()` system calls, indicates the number of
-files existing under the directory.
+The size of the directories is indicated by the `st_size` field of `struct
+stat`, which is obtained with the `stat()` or `fstat()` system calls,
+indicates the number of files that exist under the directory.
 
 ### Zone files
 
-Zone files are named using the number of the zone they represent within the set
-of zones of a particular type. That is, both the "cnv" and "seq" directories
+Zone files are named using the number of the zone they represent within the
+set of zones of a particular type. Both the "cnv" and "seq" directories
 contain files named "0", "1", "2", ... The file numbers also represent
-increasing zone start sector on the device.
+increasing zone start sector (TODO=define "increasing zone start sector") on
+the device.
```

Comment:
increasing zone start sector means that the start sector of the zone for file 0 is lower that the one for file 1 etc. File names = zone number in order of the zone report from the drive, and that order is zone start LBA increasing from 0 to max LBA. Sector 0 is the super block, then zone 1 starts at Zs for file 0, 2xZs for file 1, 3xZs for file 2, etc...

Reply:
I think that I'm going to regard this as something for my own edification instead of something that cries out for mention at this point in the text.

I do very much appreciate this explanation.